### PR TITLE
11255 general collateral editor

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#11255

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-api/migrations/versions/0bef490ea1d0_parties_split_columns.py [hunk 1]
```diff
@@ -52,13 +50,20 @@ def downgrade():
...
     op.drop_column('parties', 'first_name_split1')
     op.drop_column('parties', 'last_name_split3')
     op.drop_column('parties', 'last_name_split2')
     op.drop_column('parties', 'last_name_split1')
 
+    op.drop_column('parties', 'first_name_char1')
+    op.drop_column('parties', 'first_name_char2')
+    op.drop_column('parties', 'first_name_key_char1')
+    op.drop_column('parties', 'bus_name_base')
+    op.drop_column('parties', 'bus_name_key_char1')
```
Comment (kialj876) [inline ppr-api/migrations/versions/0bef490ea1d0_parties_split_columns.py:67]: hey steph, just verifying you meant to keep the changes in this migration?

It does look correct with this change

File: ppr-api/test_data/postgres_data_files/test0001.sql [hunk 2]
```diff
@@ -37,7 +37,7 @@ INSERT INTO parties(id, party_type, registration_id, financing_id, registration_
                   middle_initial, last_name, business_name, birth_date, address_id,
                   business_srch_key, bus_name_base, bus_name_key_char1)
     VALUES(200000002, 'DB', 200000000, 200000000, null, null, null, null, null, 'TEST BUS 2 DEBTOR',
-           null, 200000002, searchkey_business_name('TEST BUS 2 DEBTOR'), business_name_strip_designation('TEST BUS 2 DEBTOR'), 'T')
```
Comment (kialj876) [inline ppr-api/test_data/postgres_data_files/test0001.sql:40]: why the change here? I think the `business_name_strip_designation` is how the code inserts it in so it should be done here as well right?

File: ppr-api/test_data/postgres_data_files/test0002.sql [hunk 3]
```diff
@@ -24,7 +24,7 @@ INSERT INTO parties(id, party_type, registration_id, financing_id, registration_
                   middle_initial, last_name, business_name, birth_date, address_id,
                   business_srch_key, bus_name_base, bus_name_key_char1)
     VALUES(200000006, 'DB', 200000001, 200000001, null, null, null, null, null, 'TEST BUS 2 DEBTOR',
-           null, 200000003, searchkey_business_name('TEST BUS 2 DEBTOR'), business_name_strip_designation('TEST BUS 2 DEBTOR'), 'T')
```
Comment (kialj876) [inline ppr-api/test_data/postgres_data_files/test0002.sql:27]: same here?

**Review by kialj876 — COMMENTED** [review]
Hey steph this looks good I just had a couple questions

File: ppr-api/migrations/versions/0bef490ea1d0_parties_split_columns.py (hunk 1 above)
Comment (jordiwes) [reply ppr-api/migrations/versions/0bef490ea1d0_parties_split_columns.py:67]: Yeah I ran into this when I was trying to get the database going

File: ppr-api/test_data/postgres_data_files/test0001.sql (hunk 2 above)
Comment (jordiwes) [reply ppr-api/test_data/postgres_data_files/test0001.sql:40]: When I was running this, business_name_strip_designation did not exist

File: ppr-api/test_data/postgres_data_files/test0002.sql (hunk 3 above)
Comment (jordiwes) [reply ppr-api/test_data/postgres_data_files/test0002.sql:27]: same here but happy to take out

**Review by kialj876 — APPROVED** [review]
lgtm 👍
